<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign</class>
 <widget class="QWidget" name="Sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>320</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录/注册</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <widget class="QPushButton" name="AcceptButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="QuitButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTabWidget" name="SignTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="SignIn">
        <attribute name="title">
         <string>登录</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="UUsername">
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="UPassword">
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="SignUpPasswprd"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="SignUpUsername"/>
           </item>
           <item row="1" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="SignUp">
        <attribute name="title">
         <string>注册</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
           <item row="4" column="0">
            <widget class="QLabel" name="IEmail">
             <property name="text">
              <string>邮箱</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="IPassword">
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="SignInUsername"/>
           </item>
           <item row="5" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="SignInEmail"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="SignInPassword"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="IUsername">
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>Sign</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
